### PR TITLE
Update Docker image instructions

### PR DESCRIPTION
**Pull request by abitrolly**
Leave only relevant details and mention how to specify Ubuntu version

Addresses https://github.com/snapcore/snapcraft/pull/2673#issuecomment-813871624

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

**Review by sergiusens — APPROVED** [review]

File: docker/README.md [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
 
-## Setup
+By default the `Dockerfile` builds Ubuntu 16 (xenial) image with `snapcraft` from the `edge` channel.
 
-    sudo snap install docker
+    docker build . --no-cache
 
-Ensure networking works, in some cases `dockerd` might need to have something
-like `--dns=8.8.8.8` added to the command.
+It is however possible to choose base Ubuntu version and one of these channels (risk levels):
```
Comment (cjp256) [inline docker/README.md:7]:
```suggestion
It is however possible to choose the base Ubuntu version and the Snapcraft channel (risk levels):
```

File: docker/README.md [hunk 2]
```diff
@@ -1,27 +1,16 @@
 # Creating docker containers for snapcraft
 
-## Setup
+By default the `Dockerfile` builds Ubuntu 16 (xenial) image with `snapcraft` from the `edge` channel.
```
Comment (cjp256) [inline docker/README.md:3]:
```suggestion
By default the `Dockerfile` builds Ubuntu 16.04 (Xenial) image with `snapcraft` from the `edge` channel.
```

**Review by cjp256 — APPROVED** [review]
With or without the suggestions, +1

**Comment by abitrolly** [comment]
Rebased.

**Pull request merged**